systers_portal/static/css/style.css

```diff
@@ -234,6 +234,22 @@ table.decoration-none  tr a {
   font-size: 1.3em;
 }
 
+/* Meetup Location CSS
+------------------------------------------------- */
+.box-boundary{
+  outline: 1px solid black;
+}
+
+.box-container{
+  margin: 10px 40px 10px;
```

Comment:
10px for top and bottom margin seems a bit too small of a value. Especially considering that left and right margin has 40px. Try to balance it a bit and give more space to the top and bottom and less space on the sides.